src/3DMath.cpp

```diff
@@ -88,16 +88,16 @@ void Normalize(float v[3])
 #endif // WIN32_ASM
 }
 
-void InverseTransformVectorNormalize2x(float src0[3], float src1[3], float dst0[3], float dst1[3], float mtx[4][4] )
+void InverseTransformVectorNormalize2x(float src[2][3], float dst[2][3], float mtx[4][4] )
 {
-	InverseTransformVectorNormalize(src0, dst0, mtx);
-	InverseTransformVectorNormalize(src1, dst1, mtx);
+	InverseTransformVectorNormalize(&src[0], &dst[0], mtx);
```

Comment:
Compilation error:

3DMath.cpp:93: error: cannot convert 'float (*)[3]' to 'float*' for argument '1' to 'void InverseTransformVectorNormalize(float*, float*, float (*)[4])'
  InverseTransformVectorNormalize(&src[0], &dst[0], mtx);
                                                       ^